### ISSUE TITLE
Requirements.txt

### ISSUE THREAD

**Issue by nils-n** [rockNroll87q/apparent_motion#1]
Hey Michele, 
In case you are using a virtual environment (i.e. venv) to run the stimulation - would it be possible to add the requirements.txt to the main folder? 

i.e. by just running pip freeze > requirements.txt

**Comment by rockNroll87q** [comment]
Hi @nils-n , i'm not using a vm, because I don't know which version we have at the scanner. But I can do a requirements anyway.

**Comment by nils-n** [comment]
Yeah that would be nice - i just would likle to run some tests now. 
Tomorrow I'll check at the scanner which version is running there currently (for future reference)

**Comment by rockNroll87q** [comment]
added
